internal/commands/log.go

```diff
@@ -0,0 +1,169 @@
+package commands
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/urfave/cli/v3"
+	"go.uber.org/zap"
+
+	"github.com/hathora/ci/internal/commands/altsrc"
+	"github.com/hathora/ci/internal/output"
+	"github.com/hathora/ci/internal/sdk"
+	"github.com/hathora/ci/internal/setup"
+	"github.com/hathora/ci/internal/workaround"
+)
+
+var (
+	minTailLines = 1
+	maxTailLines = 5000
+)
+
+var Log = &cli.Command{
+	Name:  "log",
+	Usage: "view live process logs",
+	Flags: subcommandFlags(followFlag, processIDFlag, tailLinesFlag),
+	Action: func(ctx context.Context, cmd *cli.Command) error {
+		log, err := ProcessLogConfigFrom(cmd)
+		if err != nil {
+			//nolint:errcheck
+			cli.ShowSubcommandHelp(cmd)
+			return err
+		}
+
+		if err := log.Validate(); err != nil {
+			//nolint:errcheck
+			cli.ShowSubcommandHelp(cmd)
+			return err
+		}
+
+		log.Log.Debug("Getting build server logs...")
+
+		res, err := log.SDK.LogV1.GetLogsForProcess(
+			ctx,
+			log.ProcessID,
+			log.AppID,
+			sdk.Bool(log.Follow),
+			sdk.Int(log.TailLines))
+
+		if err != nil {
+			return fmt.Errorf("failed to get logs: %w", err)
+		}
+
+		err = output.StreamOutput(res.Stream, os.Stderr)
+		if err != nil {
+			zap.L().Error("failed to stream output to console", zap.Error(err))
+		}
+
+		return nil
+	},
+}
+
+func logFlagEnvVar(name string) string {
+	return logFlagEnvVarPrefix + name
+}
+
+var (
+	logFlagEnvVarPrefix = globalFlagEnvVarPrefix + "LOG_"
+
+	followFlag = &cli.BoolFlag{
+		Name: "follow",
+		Sources: cli.NewValueSourceChain(
+			cli.EnvVar(logFlagEnvVar("FOLLOW")),
+			altsrc.ConfigFile(configFlag.Name, "log.follow")),
+		Usage:      "to streams logs in real time",
+		Value:      false,
+		Category:   "Log:",
+		Persistent: true,
+	}
+
+	processIDFlag = &cli.StringFlag{
+		Name:    "process-id",
+		Aliases: []string{"p"},
+		Sources: cli.NewValueSourceChain(
+			cli.EnvVar(logFlagEnvVar("PROCESS_ID")),
+			altsrc.ConfigFile(configFlag.Name, "log.process-id"),
+		),
+		Category: "Log:",
+		Usage:    "system generated unique identifier to a runtime instance of your game server",
```

Comment:
if you backtick a text token in the usage string, urfave will use that value as the placeholder in help text, so I recommend we use:
```
`<id>` of the runtime instance of your game server
```

which will give the help text:
```
--process-id <id>, -p <id>      <id> of the runtime instance of your game server [$HATHORA_LOG_PROCESS_ID]
```